### DISCUSSION TITLE
github mais scenum

### DISCUSSION THREAD

**Discussion by rgcontigencia** [community/community#42159] [New to GitHub]
### Select Topic Area

Question

### Body

alguem aqui utiliza o github para postar fotos no scenum?

minhas fotos não estão subindo mesmo eu colocando o caminho dentro do scenum

subi elas no IMGBB e colei o link dentro do txt na github, esse e o formato de link está certo?

https://i.ibb.co/mH8C7B2/FOTO-01-3.jpg

**Comment by getlost01** [comment]
Parece que você está tentando exibir imagens em uma página da Web hospedada no GitHub, mas as imagens não estão aparecendo. Aqui estão algumas coisas que você pode tentar para solucionar esse problema:

1. Verifique se os links das imagens estão corretos e se as imagens estão hospedadas em um site de acesso público. Você pode verificar os links copiando e colando-os em um navegador. Se as imagens não estiverem carregando, pode haver um problema com os links ou com o site de hospedagem.
2. Verifique o caminho do arquivo para certificar-se de que está correto. Se você estiver vinculando a uma imagem armazenada em seu próprio computador, será necessário fazer o upload da imagem para um site de acesso público ou para o GitHub antes que ela possa ser exibida na página da web.
3. Certifique-se de que o arquivo de imagem esteja em um formato compatível com os navegadores da web. Alguns formatos de imagem comuns suportados pela maioria dos navegadores incluem JPEG, PNG e GIF.
4. Tente limpar o cache do navegador e recarregar a página. Às vezes, o navegador pode estar armazenando em cache versões antigas da página e não carregando a versão atualizada com as imagens.
